Example/AccessibilitySnapshot/SwiftUIView.swift

```diff
@@ -77,8 +77,27 @@ struct SwiftUIView: View {
                 .accessibility(label: Text("Label"))
                 .accessibility(value: Text("Value"))
                 .accessibility(hint: Text("Hint"))
-
-            Spacer()
```

Comment:
This removal of the spacer is notable in that it causes changes in all snapshots of this view by centering the contents however its inclusion causes the containing VStack to exceed 10 elements.

Reply:
Ugh arbitrary SwiftUI limitations...

Reply:
Can we not use `Group` here to group the circles together and retain the spacer?